Gruntfile.js

```diff
@@ -299,9 +299,15 @@ module.exports = function( grunt ) {
 							'	}\n' +
 							'} );\n';
 
+		const main =
+			"import index from './index.js';\n"+
```

Comment:
I think we ought to just make `index` the entry point of the module, to avoid pulling in `example` in environments where tree shaking is not supported. It seems fine that a consumer of example would have to go the extra mile to import from `social-logos/example` (or even `social-logos/build/example`.